app/javascript/admin/controllers/data_update_script_controller.js

```diff
@@ -4,6 +4,8 @@ export default class DataUpdateScriptController extends Controller {
   forceRun(event) {
     event.preventDefault();
     const id = event.target.dataset.value;
+    const forceRunPath = event.target.dataset.forceRunPath;
+    const indexPath = event.target.dataset.indexPath;
```

Comment:
you can use Stimulus values for this: https://stimulus.hotwire.dev/reference/values

Reply:
I've updated these, its' much cleaner with the path, but it could still use some work because we can pass through other values via stimulus functionality. It needs some refactoring. Adding a card for this well.